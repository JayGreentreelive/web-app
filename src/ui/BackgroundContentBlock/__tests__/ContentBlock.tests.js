import React from 'react'
import {Block, Media} from '@christfellowshipchurch/web-ui-kit'
import renderer from "react-test-renderer"

describe("ContentBlock Renderer", () => {
  it("Displays ContentBlock with Media displaying in the background", () => {
    const tree = renderer.create(
      <Media imageUrl="" imageAlt="">
        <Block>
          <Block.Title />
          <Block.Subtitle />
          <Block.Body />
        </Block>
      </Media>
    )
    expect(tree).toMatchSnapshot()
  })

  it("Displays ContentBlock with normal layout", () => {
    const tree = renderer.create(
      <div className="container">
        <div className="row">
          <Block imageUrl='' imageAlt=''>
            <Block.Subtitle />
            <Block.Title />
            <Block.Body />
          </Block>
        </div>
      </div>
    )
    expect(tree).toMatchSnapshot()
  })
})
