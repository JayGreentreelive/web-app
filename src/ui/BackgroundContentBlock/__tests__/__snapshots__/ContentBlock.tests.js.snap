// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentBlock Renderer Displays ContentBlock with Media displaying in the background 1`] = `
<div
  className="embed-responsive embed-responsive-1by1  "
>
  <div
    className="fill d-flex justify-content-center align-items-center"
  >
    <div
      className="container-fluid "
    >
      <div
        className="row align-items-center"
      >
        <div
          className="col-12 col-md-12 order-first px-3"
        />
        <div
          className="col-12 col-md text-center px-3"
        >
          <h2 />
          <h6 />
          <div />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ContentBlock Renderer Displays ContentBlock with normal layout 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="container-fluid "
    >
      <div
        className="row align-items-center"
      >
        <div
          className="col-12 col-md-12 order-first px-3"
        />
        <div
          className="col-12 col-md text-center px-3"
        >
          <h6 />
          <h2 />
          <div />
        </div>
      </div>
    </div>
  </div>
</div>
`;
